spec/03_find_frequency_string_in_sentence_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Specification for RubyMonk Primer Problem 3 Find the frequency of a string in a sentence
+require_relative '../03_find_frequency_string_in_sentence'
+
+RSpec.describe FindFrequencyStringInSentence do
+  subject(:string_in_sentence) { described_class.new }
+
+  describe '#find_frequency' do
+    it '"Do or do not... there is no try." has two "do"s' do
```

Comment:
I won't keep giving the same feedback; I think you get it.  Unit tests, much like higher level tests, provide the most value when they are at a higher level, and provide an general description of the behavior one can expect.  While it can be helpful to have very granular examples while TDDing, a higher-level description is a must IMO.

To put it another way, if you had no idea what this method did, would these specs be as helpful as, say, `it 'is the number of times the given word appears in the sentence'`?

Reply:
Again, agreed, but this is the RubyMonk test (description and test data).

Reply:
Aaaah sorry 'bout that, then.